pkg/cmd/attestation/api/client.go

```diff
@@ -130,6 +167,70 @@ func (c *LiveClient) getAttestations(url, name, digest string, limit int) ([]*At
 	return attestations, nil
 }
 
+func (c *LiveClient) fetchBundlesByURL(attestations []*Attestation) ([]*Attestation, error) {
```

Comment:
minor nitpick re: naming,

if the func arg is an _attestation_ what if?

```suggestion
func (c *LiveClient) fetchAttestationBundle(attestations []*Attestation) ([]*Attestation, error) {
```

Reply:
or

```suggestion
func (c *LiveClient) fetchBundleFromAttestations(attestations []*Attestation) ([]*Attestation, error) {
```